docs/reference/commandline/service_create.md

```diff
@@ -136,6 +136,130 @@ $ docker service create \
 For more information about labels, refer to [apply custom
 metadata](../../userguide/labels-custom-metadata.md).
 
+### Volumes and bind-mounts (--mount)
+
+Services use a more advanced notation to 
+
+| Option                | Required          | Description
+|:----------------------|:------------------|:--------------------------------------------------------------------------------------------------------------------
+| **type**              |                   | The type of mount, can be either "bind", to bind-mount a directory or file from the host, or "volume" to use a managed volume. Defaults to "volume" if no type is specified.
+| **src**               | depends on type   | In case of a bind-mount (`"type=bind"`), this option is required, and specifies the path on the host that is bind mounted (e.g., `src=/path/on/host/`). When using a bind-mount (`"type=bind"`), the `src` path must be specified as an absolute path, and *must* be a pre-existing path, or an error is produced.<br /><br />For volumes (`"type=volume"`), this option is optional, and specifies the name of the volume (e.g., `src=my-volume`). If a volume with the given name does not exist, it is automatically created. If this option is omitted for a volume (`type=volume`), an ephemeral volume with a random name is generated. Random names are guaranteed to be unique on the host, but may not be unique cluster-wide. Ephemeral volumes have the same lifecycle as the container it is attached to, and are destroyed when the *container* is destroyed (which is upon `service update`, or when scaling or re-balancing the service).
+| **source**            |                   | Alias for `src`.
+| **dst**               | yes               | Mount path inside the container, for example `/some/path/in/container/`. If the path does not exist in the container's filesystem, a directory is created at the specified location before the volume or bind-mount is mounted.
+| **destination**       |                   | Alias for `dst`.
+| **target**            |                   | Alias for `dst`.
+| **readonly**          |                   | Mounts the bind or volume read-only in the container. A value is optional, and can be either `true` (or `1`), or `false` (or `0`). If this option is provided without a value, then `true` is used as a default (`readonly=true` is equivalent to `readonly`).
```

Comment:
maybe:

By default, the Engine mounts binds and volumes `read-write`. Pass the `readonly` option to mount the bind or volume read-only in the container.

Value is optional
`true` or `1`: default if you don't provide a value. Mounts the bind or volume read-only in the container.
`false` or `0`: Mounts the bind or volume read-write in the container.

(this winds up being almost as wordy...up to you)